week-5/A-fetch/index.html

```diff
@@ -15,14 +15,31 @@
     text-align: center;
     margin: 0 50px;
     padding: 30px;
+    float: left;
+  }
+  button {
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+  }
+  #btn{
+    background-color: #4CAF50; /* Green */
+  }
+  #clear{
+    background-color: #ccc; /* Green */
```

Comment:
I don't think `ccc` is green :slightly_smiling_face: .

Reply:
Yeah, forgot that to change 👍